src/platform/packages/private/kbn-import-resolver/src/import_resolver.ts

```diff
@@ -142,6 +142,11 @@ export class ImportResolver {
       return this.adaptReq('src/core/server', dirname);
     }
 
+    if (req.startsWith('@modelcontextprotocol/sdk')) {
+      const relPath = req.split('@modelcontextprotocol/sdk')[1];
+      return Path.resolve(REPO_ROOT, `node_modules/@modelcontextprotocol/sdk/dist/esm/${relPath}`);
+    }
+
```

Comment:
cc @elastic/kibana-operations : this is the workaround we used with @delanni to resolve the importer errors for that package. Are we good merging it that way or do you want to implement something more "generic"?

Reply:
https://github.com/elastic/kibana/issues/216191